### PR TITLE
making requests working

### PR DESCRIPTION
**Pull request by SimonGlancy**
here is some code with which requests work

### CODE REVIEW

File: app/controllers/requests.rb [hunk 1]
```diff
@@ -0,0 +1,23 @@
+class MakersBnB < Sinatra::Base
+
+post '/requests/new' do
+  p params
```
Comment (shaneoston72) [inline app/controllers/requests.rb]: are these here for testing purposes? probably should be removed before we merge to master.

File: app/controllers/requests.rb [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+
+
+
+  params.each_pair do |k,v|
+
+    Request.create(availabledate_id: v, user_id: current_user.id, status: 'open')
+
+  end
+
+  redirect '/'
```
Comment (shaneoston72) [inline app/controllers/requests.rb]: should this route redirect to '/'? should it not route to the page that displays requests?

File: app/controllers/spaces.rb [hunk 3]
```diff
@@ -2,7 +2,8 @@ class MakersBnB < Sinatra::Base
 
 
   get '/spaces' do
-
+    puts "this is a test"
```
Comment (shaneoston72) [inline app/controllers/spaces.rb]: for testing? this probably should not be here when we merge.

File: app/controllers/spaces.rb [hunk 4]
```diff
@@ -45,7 +45,7 @@ class MakersBnB < Sinatra::Base
   get '/spaces/:id' do
     p params[:id]
```
Comment (shaneoston72) [inline app/controllers/spaces.rb]: another p....

File: app/controllers/spaces.rb [hunk 5]
```diff
@@ -2,7 +2,8 @@ class MakersBnB < Sinatra::Base
 
 
   get '/spaces' do
-
+    puts "this is a test"
+    p current_user
```
Comment (shaneoston72) [inline app/controllers/spaces.rb]: p...

**Comment by shaneoston72** [comment]
in general, really impressive effort.  especially the bit concocted by Pete to return available spaces in search results. brilliant!  

i do not think we should merge this issue's branch until it is refactored and, more importantly, add tests for functionality that is not tested.

**Comment by MuriloDalRi** [comment]
Closed #26

**Pull request closed without merging**